TruckMuncher/LoginViewController.swift

```diff
@@ -102,9 +103,13 @@ class LoginViewController: UIViewController, FBLoginViewDelegate {
     }
     
     func successfullyLoggedInAsTruck() {
-        self.dismissViewControllerAnimated(true, completion: { () -> Void in
-            NSNotificationCenter.defaultCenter().postNotificationName("loggedInNotification", object: self, userInfo: nil)
-        })
+        mgr.getFullMenus(atLatitude: 0, longitude: 0, includeAvailability: true, success: { (response) -> () in
+            self.dismissViewControllerAnimated(true, completion: { () -> Void in
+                NSNotificationCenter.defaultCenter().postNotificationName("loggedInNotification", object: self, userInfo: nil)
+            })
+        }) { (error) -> () in
+            println("error fetching full menus \(error)")
```

Comment:
if this call fails, you get stuck with nothing happening on the UI. Need to show an alert dialog.